reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -435,25 +435,25 @@ <h2 id="rule-violation-summary">Rule Violation Summary (1 errors, 1 warnings, 1
       <table class="ort-report-table ort-violations">
         <thead>
           <tr>
-            <th>#</th><th>Source</th><th>Violation</th>
+            <th>#</th><th>Rule</th><th>Package</th><th>License</th><th>Message</th>
           </tr>
         </thead>
         <tbody>
           <tr class="ort-error" id="violation-1">
-            <td><a href="#violation-1">1</a></td><td>GPL-1.0-only</td><td>
-              <p>ERROR: GPL 1 error</p>
+            <td><a href="#violation-1">1</a></td><td><REPLACE_TIMESTAMP></td><td>Maven:junit:junit:4.12</td><td>DETECTED: EPL-1.0</td><td>
```

Comment:
Aren't the timestamps gone now from here?

Reply:
Fixed.